assembl/static2/css/components/posts.scss

```diff
@@ -186,6 +197,14 @@ $deleted-post-border: 1px solid $light-grey !default;
 }
 
 .posts {
+  .post-body-title {
+    font-family: $font-family;
+    text-transform: uppercase;
+    font-size: 17px;
+    line-height: 24px;
```

Comment:
line height should be calculated accorded to the font size and not hardcoded

Reply:
maybe there is a line-height already define among the variables

Reply:
Actually I don't need a line-height here, I deleted it.